### PR TITLE
Run the manual partitioning task for the given requests

### PR DESCRIPTION
**Pull request by poncovka**
Provide the list of requests as an argument of the task.
Don't retrieve them in the task using DBus.

### CODE REVIEW

File: pyanaconda/modules/storage/partitioning/manual_partitioning.py [hunk 1]
```diff
@@ -34,19 +32,25 @@
...
     """A task for the manual partitioning configuration."""
 
+    def __init__(self, storage, requests):
+        """Create a task.
+
+        :param storage: an instance of Blivet
+        :param requests: a list of requests
+        """
+        super().__init__(storage)
+        self._requests = list(requests)
```
Comment (jkonecny12) [inline pyanaconda/modules/storage/partitioning/manual_partitioning.py]: If it should be list (based on the doc string) why are you making a list from the `requests` variable?

File: pyanaconda/modules/storage/partitioning/manual_partitioning.py [hunk 2]
```diff
@@ -34,19 +32,25 @@
...
-        manual_part_proxy = STORAGE.get_proxy(MANUAL_PARTITIONING)
-
-        # Set up mount points.
-        for mount_data in manual_part_proxy.Requests:
-            self._setup_mount_point(storage, MountPointRequest.from_structure(mount_data))
+        for mount_data in self._requests:
+            self._setup_mount_point(storage, mount_data)
 
-    def _setup_mount_point(self, storage, mount_data):
+    def _setup_mount_point(self, storage, mount_data: MountPointRequest):
```
Comment (jkonecny12) [inline pyanaconda/modules/storage/partitioning/manual_partitioning.py]: It's already in the documentation, what is the reason to specify it here for a protected method?

File: pyanaconda/modules/storage/partitioning/manual_partitioning.py (hunk 1 above)
Comment (poncovka) [reply pyanaconda/modules/storage/partitioning/manual_partitioning.py]: Fixed.

File: pyanaconda/modules/storage/partitioning/manual_partitioning.py (hunk 2 above)
Comment (poncovka) [reply pyanaconda/modules/storage/partitioning/manual_partitioning.py]: Fixed.

**Comment by poncovka** [comment]
Updated.

**Pull request merged**